### ISSUE TITLE
Release a new Pundit version with recent changes?

### ISSUE THREAD

**Issue by mullermp** [varvet/pundit#518]
Our company does source import of gems through rubygems. It looks like some changes have been made since the last official release, and I'm unable to get those changes until it goes through the appropriate published channels.

In particular, the README states that the #authorize method can be chained:

    authorize returns the object passed to it, so you can chain it like this:
    def show
        @user = authorize User.find(params[:id])
    end

But the actual docs for version 1.1.0 do not align with that:

https://www.rubydoc.info/gems/pundit/1.1.0/Pundit#authorize-class_method

This change was made about two years ago but hasn't been bundled into a new version:

https://github.com/varvet/pundit/commit/21d4c1bd2d0a2d69b8c32653870cd63bf0ca0eae#diff-43022d52f798af4abc72a2783a87cd51

Is it at all possible that Pundit can be bumped and released to version 1.1.1 or 1.2? Importing through other means is a no go. Our company is very large and has strict infosec policies.

**Comment by Linuus** [comment]
Pekade see: https://github.com/varvet/pundit/issues/515

TLDR: I’m working on it :)

**Comment by mullermp** [comment]
What's the ETA on that? Can we get a minor version before 2.0? That seems like a heavy handed change.

**Comment by Linuus** [comment]
@mullermp No ETA, sorry. But I hope it won't be long.

Well, changing the return value is a breaking change and thus requires us to release a new major version. So a minor won't do with what's in master at the moment.

I just want to get a few other fixes in there before the release. I don't want to end up releasing 2.0 and realize I need another breaking change and be forced to release 3.0...

I don't expect 2.0 to be a big upgrade for end users though.

**Comment by mullermp** [comment]
That makes sense. Thanks for the update. For now I'll call 'authorize' without using the return value.

**Comment by Linuus** [comment]
You're welcome :)
